web-basics-browser-security-model/README.md

```diff
@@ -0,0 +1,403 @@
+# Introduction
+
+[Session 01 slides](https://docs.google.com/presentation/d/19ntegcXmtk2DGBfr-FMBvEVTitvqaxaOROKlDUnNhEM/edit?usp=sharing)
```

Comment:
Can we also upload the slides in PDF format, in a `slides/` directory, as instructed [here](https://github.com/security-summer-school/template-public)?